### PR TITLE
🧑‍💻 ToString: improve test and bench setup

### PR DESCRIPTION
**Pull request by ReneWerner87**
- add string slice case
- add parallel instruction for child tests
- benchmark table and parallel execution

### CODE REVIEW

**Comment by ReneWerner87** [comment]
image

File: convert_test.go [hunk 1]
```diff
@@ -136,10 +144,19 @@ func Benchmark_ToString(b *testing.B) {
...
+		[]int{42, 21},
+		[2]int{42, 21},
+		[]interface{}{[]int{42, 21}, 42, "Hello", true, []string{"Hello", "World"}},
 	}
-	for n := 0; n < b.N; n++ {
-		for _, value := range values {
-			_ = ToString(value)
-		}
+	for _, value := range values {
+		b.Run(reflect.TypeOf(value).String(), func(b *testing.B) {
```
Comment (gaby) [inline convert_test.go:153]: RunParallel should be a separate benchmark funct. That way we can compare between sequential benchmark and concurrent

**Review by gaby — COMMENTED** [review]
LGTM so far

File: convert_test.go [hunk 2]
```diff
@@ -93,9 +93,13 @@ func Test_ToString(t *testing.T) {
 		{time.Date(2000, 1, 1, 12, 34, 56, 0, time.UTC), "2000-01-01 12:34:56"},
 		{struct{ Name string }{"John"}, "{John}"},
 		{[]string{"Hello", "World"}, "[Hello World]"},
+		{[]int{42, 21}, "[42 21]"},
+		{[2]int{42, 21}, "[42 21]"},
+		{[]interface{}{[]int{42, 21}, 42, "Hello", true, []string{"Hello", "World"}}, "[[42 21] 42 Hello true [Hello World]]"},
```
Comment (efectn) [inline convert_test.go:98]: can you add [][]int testcase

File: convert_test.go (hunk 1 above)
Comment (ReneWerner87) [reply convert_test.go:171]: https://github.com/gofiber/utils/pull/63/commits/e00da2ac657e73d44e9822a72e943a24d70cbaac

File: convert_test.go (hunk 2 above)
Comment (ReneWerner87) [reply convert_test.go:127]: https://github.com/gofiber/utils/pull/63/commits/fb9aac5b7b2b1acb8146ae3445fcd33f811f3a04

**Review by gaby — APPROVED** [review]
LGTM

**Review by efectn — APPROVED** [review]

**Pull request merged**